enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -122,3 +122,12 @@ def _get_course_run_url(self, request, course_run_id):
                 course_run_url = '{}?{}'.format(exec_ed_base_url, urlencode(params))
 
         return course_run_url
+
+    def _get_resume_course_run_url(self, course_run_id, request):
```

Comment:
[suggestion] It may be worth adding an assertion and/or test case that the courses.edx.org domain is included in the `resume_course_run_url` field.